e2e/cypress/integration/dashboard-transloadit.spec.ts

```diff
@@ -53,67 +53,73 @@ describe('Dashboard with Transloadit', () => {
   it('should emit one assembly-cancelled event when cancelled', FLAKY, () => {
```

Comment:
If it's no longer flaky, let's remove this :))
```suggestion
  it('should emit one assembly-cancelled event when cancelled', () => {
```

Reply:
I'm keeping the FLAKY for now until these changes prove themselves. It's mostly an assumption after all, and it doesn't hurt to have it.

Reply:
> it doesn't hurt to have it.

I'm not sure I agree, if we keep it around chances are it's going to be harder to detect when another PR introduces flaky tests, which would decrease the quality of the e2e test results again. I suggest we remove it, and then we do a stress test: run the test again and again like 20 times in a row and see if there are any flakiness left.